nix/.stack.nix/cs-blockchain.nix

```diff
@@ -39,6 +39,7 @@
             (hsPkgs.tasty-hunit)
             (hsPkgs.tasty-hedgehog)
             (hsPkgs.cs-blockchain)
+            (hsPkgs.cs-ledger)
```

Comment:
uh? Wasn't this supposed to be automatically generated?

Reply:
Yes, I regenerated things as part of this PR